.github/workflows/nightlies.yml

```diff
@@ -23,8 +23,8 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-          - '3.8'
           - '3.9'
+          - '3.10'
```

Comment:
It might make sense to set these to `3.12`, so that the oldest and newest Python version are both tested